<<<<<<< HEAD
Disable STL wrappers in some places to help libc++ 9

diff --git config/external/rlbox/moz.build config/external/rlbox/moz.build
index d216e62cad861..391bbdf566b88 100644
--- config/external/rlbox/moz.build
+++ config/external/rlbox/moz.build
@@ -26,4 +26,7 @@ SOURCES += [
     'rlbox_thread_locals.cpp'
 ]
 
+# bug 1594027
+DisableStlWrapping()
+
 FINAL_LIBRARY = 'xul'
\ No newline at end of file
diff --git gfx/graphite2/src/moz.build gfx/graphite2/src/moz.build
index 447cd861126a3..8a4fcd814e6fb 100755
--- gfx/graphite2/src/moz.build
+++ gfx/graphite2/src/moz.build
@@ -69,6 +69,9 @@ if CONFIG['MOZ_WASM_SANDBOXING_GRAPHITE']:
 # thebes
 DefineAndWasmDefine('GRAPHITE2_STATIC', True)
 
+# bug 1594027
+DisableStlWrapping()
+
 # We allow warnings for third-party code that can be updated from upstream.
 AllowCompilerWarnings()
 
=======
commit 9949bb054e7a
Author: Greg V <greg@unrelenting.technology>
Date:   Mon Feb 24 20:18:01 2020 +0000

    Bug 1594027 - add more STL wrappers to fix build with libc++ 9 r=froydnj
    
    Differential Revision: https://phabricator.services.mozilla.com/D62847
    
    --HG--
    extra : moz-landing-system : lando
---
 config/gcc-stl-wrapper.template.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git config/gcc-stl-wrapper.template.h config/gcc-stl-wrapper.template.h
index 41ae14024b0c0..9fe70fbaab8dc 100644
--- config/gcc-stl-wrapper.template.h
+++ config/gcc-stl-wrapper.template.h
@@ -32,6 +32,14 @@
 #  define moz_dont_include_mozalloc_for_cstdlib
 #endif
 
+#ifndef moz_dont_include_mozalloc_for_type_traits
+#  define moz_dont_include_mozalloc_for_type_traits
+#endif
+
+#ifndef moz_dont_include_mozalloc_for_limits
+#  define moz_dont_include_mozalloc_for_limits
+#endif
+
 // Include mozalloc after the STL header and all other headers it includes
 // have been preprocessed.
 #if !defined(MOZ_INCLUDE_MOZALLOC_H) && \
>>>>>>> upstream/master
